core/Pos/Core/Constants/Raw.hs

```diff
@@ -208,6 +211,11 @@ staticSysStartRaw
     | otherwise     =
           sec $ ccProductionNetworkStartTime coreConfig
 
+-- | Protocol version. When serializing items into the node's DB, the values are paired
```

Comment:
I wouldn't call it `Protocol version`. There are multiple versions in different places of `cardano-sl` which can also be called `Protocol version`. Without thinking much about naming, I can propose something like `DB format version`. The name should reflect that this version applies only to the way we serialize data into DB.